src/components/next/table/row/style.css

```diff
@@ -0,0 +1,21 @@
+:host {
+	grid-column: 1 / -1;
+	display: grid;
+	grid-template-columns: subgrid;
+	border-bottom: 1px solid rgb(var(--smoothly-color-shade));
+}
+
+:host:has(smoothly-next-table-expandable-cell) {
+	grid-template-rows: auto 0fr 0fr;
+	/*grid-auto-flow: row dense;*/
+	/* transition: all 400ms linear; */
```

Comment:
Can this be removed?

Reply:
I believe that
`grid-auto-flow: row dense;` fixed an issue with mixing expandable cells and normals cells
```
| expandable | normal | expandable | normal | expandable |
```
If a table of such structure looks ok then we can remove it iirc.